recipes/fast-dds/all/conandata.yml

```diff
@@ -2,7 +2,12 @@ sources:
   "2.3.2":
     url: "https://github.com/eProsima/Fast-DDS/archive/refs/tags/v2.3.2.tar.gz"
     sha256: "4D8183CF4D37C3DE9E6FD28D2850DD08023A9079001C4880B23C95F0D8C0B5CE"
+  "2.3.3":
+    url: "https://github.com/eProsima/Fast-DDS/archive/refs/tags/v2.3.3.tar.gz"
+    sha256: "5EBF27D810C6AB68EEF7D42937CD421D85E50509AE96883239979A1B3A2F4F82"
 patches:
   "2.3.2":
     - base_path: "source_subfolder"
       patch_file: "patches/2.3.2-0001-fix-find-asio-and-tinyxml2.patch"
+  "2.3.3":
+    - base_path: "source_subfolder"
```

Comment:
not sure what is here official process - 2.3.3 dont need patch - so i just gave the base_path but not the patch_file

Reply:
Ha, didn't know that works. Nice hack.

Anyway, the normal/official way is to don't add the version in patches and do the following in `conanfile.py`.
```python
for patch in self.conan_data.get("patches", {}).get(self.version, []):
    tools.patch(**patch)
```

Reply:
thanks for info - but i found out i have to patch again - i was missing my checks inside container .... so it took tinyxml from system .... so still have to add patch. ^^'